apps/site/assets/ts/schedule/components/schedule-finder/ScheduleModalContent.tsx

```diff
@@ -116,6 +120,7 @@ const ScheduleModalContent = ({
   return (
     <>
       <div className="schedule-finder schedule-finder--modal">
+        {/* TODO figure out where selected origin is set, maybe it isn't a stopId at its source */}
```

Comment:
Well it's always a stopId - it's set by the selected origin dropdown at top of the schedule finder. 👍🏼

Reply:
Good to know